src/components/structural/header/Header.js

```diff
@@ -553,6 +555,43 @@ class Header extends Component {
         this.setState({ snackOpen: false });
     }
 
+    onExport = (scene = undefined) => {
+        let param = "";
+        if(scene) {
+            param = `?id=${scene}`;
+        }
+
+        fetch(`/apiv1/scenes/export${param}`, {headers: {"x-access-token": this.props.user.uid}}).then(async (resp) => {
+            switch(resp.status) {
+                case 204:
+                    alert("There are no scenes saved for you to export!");
+                    break;
+                case 200:
+                    let data = await resp.json();
+                    saveAs(new Blob([JSON.stringify(data)]), "MYR-export.json", {
```

Comment:
I think having exported file to have a unique name like scene (eg. "MYR-Box.json") might be better for user to recognize what scene is it, but I can see there might be a problem if user have some obscure scene name. (Also realized that you put every scenes in single json file when choose export all)

Reply:
Also, I'm not a huge fan of putting every scene in single json file when "export all" since user might want to select what scene they want to import. Obviously, it's easy to separate scenes to individual json file but it might not be intuitive for new user. But I think it's fine for now.

Reply:
The way that I will have this working is in the "Open" dialog, it exports all the scenes (we can impelment a checkbox thing later if we want), but in the `Sharing` tab it will only export the current scene.

As for the name of the file...  I don't know of too many ways to make it more unique.. I could use the window title, but then it's not really descriptive of the scenes when the user exports all scenes.  Maybe a date appended onto the end?